capture/config.default.json

```diff
@@ -14,12 +14,14 @@
   ],
   "onBeforeScript": "puppet/onBefore.js",
   "onReadyScript": "puppet/onReady.js",
+  "localStorageData": "Does not have precedence over the values in scenarios"
   "scenarios": [
     {
       "label": "BackstopJS Homepage",
       "cookiePath": "backstop_data/engine_scripts/cookies.json",
       "url": "https://garris.github.io/BackstopJS/",
       "referenceUrl": "",
+      "localStorageData": "backstop_data/engine_scripts/localStorage.json"
```

Comment:
Please model this after the load cookies feature.   Please note the cookie param name `cookiePath`.   This should be `localStoragePath` (or something to that effect)

Reply:
I renamed scenarios[0].localStorageData to scenarios[0].localStoragePath, I removed config.localStorageData